config/session.js

```diff
@@ -17,7 +17,7 @@ module.exports.session = {
    * of your users, forcing them to log in again.                             *
    *                                                                          *
    ***************************************************************************/
-  secret: "cf1e0dd462546522d7cebec3f4b841fc"
+  secret: "cf1e0dd462546522d7cebec3f4b841fc",
```

Comment:
should we be storing this in GitHub?

Reply:
You're right. The secret should be placed in `local.js` or `.env`. And it really ought to be a unique value for every AB instance.

But can we deal with `isSessionDisabled` first? The only change I made to the referenced line was adding a comma at the end.

It's probably fine to just leave this "default" secret here. It should be overrided by local.js.